README.md

```diff
@@ -32,6 +33,7 @@ After building your image, use [push-to-registry](https://github.com/redhat-acti
 
 | Input Name | Description | Default |
 | ---------- | ----------- | ------- |
+| archs | Architecture(s) to build the image(s) for. For multiple architectures, separate by a comma. If running on self hosted runners then suggest running [qemu-user-static](https://github.com/marketplace/actions/docker-setup-qemu) action to install `qemu-user-static`. | `amd64`
```

Comment:
If the workflow is running on a self-hosted runner, [docker-setup-qemu] can be used to install the `qemu-user-static` dependency.